ballerina/Ballerina.toml

```diff
@@ -12,8 +12,8 @@ distribution = "2201.5.0"
 [[platform.java11.dependency]]
 groupId = "io.ballerina.stdlib"
 artifactId = "http-native"
-version = "2.7.3"
-path = "../native/build/libs/http-native-2.7.3.jar"
+version = "2.7.4"
+path = "../native/build/libs/http-native-2.7.4-SNAPSHOT.jar"
```

Comment:
is it intentional to change this version with the SNAPSHOT version?

Reply:
Yes, it is from the automated commit